project-koku/koku
# Cost 2511 storage currency

Fixes [COST-2511](https://issues.redhat.com/browse/COST-2511)

Changes proposed in this PR:
* adding currency support for storage report types to all providers and OCP on providers
*

Testing Instructions:
To be Added
1. Run `this command` ...
2. Ingest this data `with this other command` ...
3. Expect to see this result:
   ```
   GET http://localhost:8000/api/endpoint

   {json: response}
   ```
4.
5.

---
Additional Context:
